docs/src/operate/c8y/supported-operations.md

```diff
@@ -140,40 +140,35 @@ The `Supported Operations API` can also be used to add support for custom operat
 For e.g: Cumulocity supports the `c8y_Command` that enables execution of shell commands on the device from the cloud.
 The same operation files can be used to define how the mapper should handle these operations when triggered from the cloud.
 
-### Adding new custom operation
+### Creating new custom operation
+
+Custom operations can be delivered by SmartREST message or JSON over MQTT depending on the content of the operation file. The former is limited to SmartREST input. The latter moves the actual execution of the operation to `tedge-agent` where is executed by the workflow.
```

Comment:
I still find that this text is missing an introduction.

```suggestion
Custom operations can be triggered on the device by SmartREST message or JSON over MQTT.
In both case the behavior of thin-edge is defined by the operation configuration file that tells to which messages the device must react and how.

* With SmartREST, the SmartREST code of the message determines the operation to execute and  the SmartREST payload is passed unchanged to a script to which the command is delegated.
* With JSON over MQTT the name of operation to execute is given by the JSON payload of the message and the operation is delegated by the agent running on the target device (which can be the main device of a child device ), possibly with some transformation of the JSON payload.
```

Reply:
Resolved in 46b39e8121774672f354a4ec86017bdb715254c3